lib/assert-result-factory.js

```diff
@@ -1,33 +1,22 @@
 /** @babel */
 
-class AssertResultFactory {
-	constructor() {
-		this.groupIsolationChar = '›';
-	}
-
-	getAssertResult(assert, context) {
-		const result = {
-			assert,
-			group: '',
-			currentExecution: context
-		};
+module.exports.getAssertResult = function (assert, context) {
```

Comment:
Just export it directly:

```js
module.exports = (assert, context) => {
```

Note arrow function too.

And just rename the file to `get-assert-result.js`.